src/MailThief.php

```diff
@@ -151,4 +149,11 @@ public function alwaysFrom($address, $name = null)
     {
         $this->from = ['address' => $address, 'name' => $name];
     }
+
+    public function loadGlobalFrom()
```

Comment:
I'm not a big fan of making something a public method just for the sake of testing. It's not really a part of the public API, right?